### PR TITLE
native_extension_fuzzing.md: show steps to confirm proper linking

### PR DESCRIPTION
**Pull request by disconnect3d**
Hi,

Idk if you want such change, but I guess we can inform people how to confirm they linked the library properly. This can be useful if one is using a complex build system and is unsure whether CFLAGS/CXXFLAGS flags were passed properly :P.

### CODE REVIEW

**Comment by disconnect3d** [comment]
Example symbols for me:
```
0000000002f1ae00 d __sancov_gen_cov_switch_values.99
0000000002fa4190 d __sancov_gen_cov_switch_values.99
0000000002fb7e90 d __sancov_gen_cov_switch_values.99
0000000002f012b0 d __sancov_gen_cov_switch_values.990
0000000002f01300 d __sancov_gen_cov_switch_values.995
0000000002f01350 d __sancov_gen_cov_switch_values.996
                 U __sancov_lowest_stack
0000000002fe19b0 d __start___sancov_cntrs
0000000003024588 d __start___sancov_pcs
0000000003024582 d __stop___sancov_cntrs
00000000034502a8 d __stop___sancov_pcs
0000000000a79cc0 t sancov.module_ctor_8bit_counters
```

```
                 U __asan_after_dynamic_init
                 U __asan_before_dynamic_init
                 U __asan_handle_no_return
                 U __asan_init
                 U __asan_load_cxx_array_cookie
                 U __asan_memcpy
                 U __asan_memmove
                 U __asan_memset
                 U __asan_option_detect_stack_use_after_return
                 U __asan_poison_cxx_array_cookie
                 U __asan_register_globals
                 U __asan_report_load1
                 U __asan_report_load16
                 U __asan_report_load2
                 U __asan_report_load4
                 U __asan_report_load8
                 U __asan_report_load_n
                 U __asan_report_store1
                 U __asan_report_store16
                 U __asan_report_store2
                 U __asan_report_store4
                 U __asan_report_store8
                 U __asan_report_store_n
                 U __asan_set_shadow_00
                 U __asan_set_shadow_f5
                 U __asan_set_shadow_f8
                 U __asan_stack_free_5
                 U __asan_stack_free_6
                 U __asan_stack_free_7
                 U __asan_stack_free_8
                 U __asan_stack_free_9
                 U __asan_stack_malloc_0
                 U __asan_stack_malloc_1
                 U __asan_stack_malloc_2
                 U __asan_stack_malloc_3
                 U __asan_stack_malloc_4
                 U __asan_stack_malloc_5
                 U __asan_stack_malloc_6
                 U __asan_stack_malloc_7
                 U __asan_stack_malloc_8
                 U __asan_stack_malloc_9
                 U __asan_unregister_globals
                 U __asan_version_mismatch_check_v8
0000000001628910 t asan.module_ctor
000000000175b190 t asan.module_ctor
0000000001820550 t asan.module_dtor
```

File: native_extension_fuzzing.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ CC="/usr/bin/clang" CFLAGS="-fsanitize=address,fuzzer-no-link" CXX="/usr/bin/cla
 Here, `address` means Address Sanitizer. You can also use `undefined` for the
 Undefined Behavior Sanitizer.
 
+If you use want to confirm that the built library is compiled/linker properly, you can do it by checking if it contains asan and sancov related symbols via a command like `nm <path to .so file> | grep asan` (or `grep sancov`).
```
Comment (TheShiftedBit) [inline native_extension_fuzzing.md:17]: linked, not linker

Comment (disconnect3d) [inline native_extension_fuzzing.md:17]:
```suggestion
If you use want to confirm that the built library is compiled/linked properly, you can do it by checking if it contains asan and sancov related symbols via a command like `nm <path to .so file> | grep asan` (or `grep sancov`).
```

**Comment by jvoisin** [comment]
Can you please resolve conflicts?

**Pull request merged**